src/base64.h

```diff
@@ -10,8 +10,9 @@
 
 namespace node {
 //// Base 64 ////
-#define base64_encoded_size(size) ((size + 2 - ((size + 2) % 3)) / 3 * 4)
-
+static inline constexpr size_t base64_encoded_size(size_t size) {
```

Comment:
`constexpr` implies `inline`, so no need to specify `inline`

Reply:
https://en.cppreference.com/w/cpp/language/constexpr says

> A constexpr specifier used in a function or static member variable (since C++17) declaration implies inline

We are not yet at C++17, right? IINW, We are still at C++14.

Reply:
CPPReference is tricky to read. It's only the `static member variable (since C++17)` part that is the addition to C++17 ([see this change](https://en.cppreference.com/mwiki/index.php?title=cpp/language/constexpr&diff=104140&oldid=89204))
In C++11 there were no `constexpr static` member variables, they were added in C++17.